src/browzine-summon-adapter.js

```diff
@@ -124,30 +131,63 @@ angular.module("summonApp.directives")
     return template;
   };
 
-  return {
-    link: function(scope, element, attributes) {
-      if(!shouldEnhance(scope)) {
-        return;
+  function documentSummary(scope, element) {
+    if(!shouldEnhance(scope)) {
+      return;
+    }
+
+    var endpoint = getEndpoint(scope);
+
+    $.getJSON(endpoint, function(response) {
+      var data = getData(response);
+      var browzineWebLink = getBrowZineWebLink(data);
+      var coverImageUrl = getCoverImageUrl(data, response);
+
+      if(browzineWebLink) {
+        var template = buildTemplate(data, browzineWebLink, bookIcon);
+        $(element).find(".docFooter .row:first").append(template);
       }
 
-      var endpoint = getEndpoint(scope);
+      if(coverImageUrl) {
+        $(element).find(".coverImage img").attr("src", coverImageUrl);
+      }
+    });
+  };
 
-      http.get(sce.trustAsResourceUrl(endpoint)).then(function(response) {
-        var data = getData(response);
-        var browzineWebLink = getBrowZineWebLink(data);
-        var coverImageUrl = getCoverImageUrl(data, response);
+  function browZineEnhance(element) {
+    var secret = Object.getOwnPropertyNames(element).filter(function(property) {
```

Comment:
The name `secret` here has me a little confused - this isn't a secret key or anything, correct?  Looks like this is the name of an element's property that is kind of hard to find because of how Angular builds a page?

Reply:
It's for this type of property `jQuery321096145541210543752`, which we only need to access the Angular $scope with all the data for the search result:

![image](https://user-images.githubusercontent.com/376230/31558376-9a19b894-b01a-11e7-88c2-6acce2eb540d.png)

Someone put in work to make it not easily accessible and I googled and couldn't find what it is exactly, `secret` was nice and short, so it worked for me. Suggestions?

Reply:
Roger that.  Maybe `propertyCreatedForBZIntegration` ?  (If I'm understanding what it's doing correctly, I think that better describes what it is)

Reply:
Was `jQuery321096145541210543752` actually created for BZ integration though? xD

Reply:
Aha, then I do not understand correctly what it's doing 😊 

I thought it was being created as a result of the BZ summon instance and was sitting somewhere in the HTML, but I'm not seeing that looking at the page now.  

`secret` or `token` is fine then, don't mind me, I just didn't fully understand what was going on. 😁 

Still kind of curious where in the code this is appearing, since I don't see it in http://msulibrariestest.summon.serialssolutions.com/#!/search?ho=t&l=en&q=new%20england%20journal%20of%20medicine , but no big deal - don't let my curiosity here hold us up from merging. 👍

Reply:
Thanks Karl! `jQuery321096145541210543752 ` seems to be buried in the jQuery/Angular internals somehow. Summon is basically getting data from their database for the search result and then storing it on Angular's $scope object. We're definitely not supposed to be using it, feels kinda naughty, hehehe.

I was only able to find it by doing a `$('.documentSummary')` and then inspecting the jQuery objects in the results. No crystal ball here, but I know we'll need to collaborate w/ the Summon team about the LABjs issue at some point, I'm just not sure when strategy-wise TBH. Some good brandy or whiskey might help shake out some thoughts though. :)

Reply:
Google says Serials Solutions HQ is in Seattle... from my little experience there, a craft beer, PBR, or [tiki drink](https://www.yelp.com/biz/lava-lounge-seattle?osq=tiki+bar) could work. 🍻  🍸